src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,44 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
+
+  this.signal('loadBindingList');
+
+  this.container.find('#genes .glyphicon-remove')
+    .css('display', this.data.tracks.length == 1 ? 'none' : '');
+};
+
+/**
+ * Updates the track list for panel.
+ */
+genotet.BindingPanel.prototype.updateTrackList = function() {
+  var fileNames = genotet.data.bindingFiles.map(function(dataInfo) {
+    return {
+      id: dataInfo.fileName,
+      text: dataInfo.gene + ' (' + dataInfo.fileName + ')'
+    };
+  });
   this.data.tracks.forEach(function(track, index) {
     var ui = this.container.find('#genes #track-' + index);
+
     if (!ui.length) {
```

Comment:
Have you tested that track removal works properly for the panel? What happens if you remove a track in the middle?